README.md

```diff
@@ -78,6 +80,19 @@ Congrats, you now have the Argus frontend up and running!
 
 Note that the website will automatically reload as you edit the code.
 
+### Using Docker Compose
+
+This repository contains a `docker-compose.yml` definition to run all the backend components as services, while the Argus frontend runs directly off the checked out source code using npm. To accomplish this, all you need should be:
+
+    export UID
+	docker-compose up
```

Comment:
Is this something you type in the shell? If so, could you make it clearer? If not, what is this? "all you need should be:" how exactly?

Reply:
Yes. I've been editing both reStructured Text docs and Markdown docs today, so apparently I got confused about the markup :-D